### ISSUE TITLE
Using observer mode should be made costly

### ISSUE THREAD

**Issue by vvahut** [airmash-refugees/game-ideas#64]
Some players idle in the observer mode after getting to the top. 
This should be prevented by decreasing player's points / updgrades every x seconds.

-- blob

**Comment by LazerSharkOverlord** [comment]
Na Mate that won't work some people actually need to go and taking points from that sincerely won't be a good idea. Maybe let's say you were at the top and you needed to do something imagine by the time you came back all your points where gone? would you really like that?

**Comment by vvahut** [comment]
> 
> 
> Na Mate that won't work some people actually need to go and taking points from that sincerely won't be a good idea. Maybe let's say you were at the top and you needed to do something imagine by the time you came back all your points where gone? would you really like that?

It's all about adjusting. You could make it so that the cost effect would begin after 5 or 10 minutes, if you're worried about the "legitimate use cases".

**Comment by LazerSharkOverlord** [comment]
> > Na Mate that won't work some people actually need to go and taking points from that sincerely won't be a good idea. Maybe let's say you were at the top and you needed to do something imagine by the time you came back all your points where gone? would you really like that?
> 
> It's all about adjusting. You could make it so that the cost effect would begin after 5 or 10 minutes, if you're worried about the "legitimate use cases".


**Yea your right I'll have a chat with a Dev. to see what we can do about it : )**

**Comment by vff-the-god** [comment]
Well sometimes people need to go eat or go to the bathroom, so i dont think so, maybe like an hour and take away but nothing less

**Comment by vvahut** [comment]
> 
> 
> Well sometimes people need to go eat or go to the bathroom, so i dont think so, maybe like an hour and take away but nothing less

Yeah, in that case you can just quit the game and come back later. But anyway, I think I made my point already.
